utils/conversions.go

```diff
@@ -285,6 +279,9 @@ func ToString(env Environment, val interface{}) (string, error) {
 		return "", nil
 	}
 
+	// if value is a variable resolver, then use it's default
+	val = ResolveDefault(val)
+
```

Comment:
As above I find this confusing.. why `Default` take precedence over `String` when trying to Stringify something?